Intermittent failure of MicroProfileHealthSupervisedRouteTest.supervisedRouteTest on Windows

### Bug description

```
org.awaitility.core.ConditionTimeoutException: 
Assertion condition defined as a Lambda expression in org.apache.camel.quarkus.component.microprofile.it.health.MicroProfileHealthSupervisedRouteTest 4 expectations failed.
JSON path status doesn't match.
Expected: is "DOWN"
  Actual: UP

JSON path checks.status.findAll().unique() doesn't match.
Expected: iterable containing ["UP", "DOWN"]
  Actual: <[UP]>

JSON path checks.find { it.name == 'camel-routes' }.status doesn't match.
Expected: is "DOWN"
  Actual: UP

JSON path checks.find { it.name == 'camel-consumers' }.status doesn't match.
Expected: is "DOWN"
  Actual: UP
 within 5 seconds.
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:1160)
	at org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:790)
	at org.apache.camel.quarkus.component.microprofile.it.health.MicroProfileHealthSupervisedRouteTest.supervisedRouteTest(MicroProfileHealthSupervisedRouteTest.java:38)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:101)
	at org.awaitility.core.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:81)
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:103)
	... 5 more
```

--- a/integration-tests/microprofile-health/src/test/java/org/apache/camel/quarkus/component/microprofile/it/health/MicroProfileHealthSupervisedRouteTest.java
+++ b/integration-tests/microprofile-health/src/test/java/org/apache/camel/quarkus/component/microprofile/it/health/MicroProfileHealthSupervisedRouteTest.java
@@ -35,7 +35,7 @@ class MicroProfileHealthSupervisedRouteTest {
     @Test
     void supervisedRouteTest() throws InterruptedException {
         // Verify the initial health state
-        await().atMost(5, TimeUnit.SECONDS).untilAsserted(() -> {
+        await().atMost(15, TimeUnit.SECONDS).untilAsserted(() -> {
             RestAssured.when().get("/q/health").then()
                     .contentType(ContentType.JSON)
                     .header("Content-Type", containsString("charset=UTF-8"))
@@ -46,7 +46,7 @@ class MicroProfileHealthSupervisedRouteTest {
         });
 
         // camel.routecontroller.unhealthy-on-exhausted is false so the heath status should be UP
-        await().pollDelay(1, TimeUnit.SECONDS).atMost(5, TimeUnit.SECONDS).untilAsserted(() -> {
+        await().pollDelay(1, TimeUnit.SECONDS).atMost(15, TimeUnit.SECONDS).untilAsserted(() -> {
             RestAssured.when().get("/q/health").then()
                     .contentType(ContentType.JSON)
                     .header("Content-Type", containsString("charset=UTF-8"))

--- a/integration-tests/microprofile-health/src/test/java/org/apache/camel/quarkus/component/microprofile/it/health/SupervisedRouteTestProfile.java
+++ b/integration-tests/microprofile-health/src/test/java/org/apache/camel/quarkus/component/microprofile/it/health/SupervisedRouteTestProfile.java
@@ -26,7 +26,7 @@ public class SupervisedRouteTestProfile implements QuarkusTestProfile {
         return Map.of(
                 "camel.routecontroller.enabled", "true",
                 "camel.routecontroller.initial-delay", "0",
-                "camel.routecontroller.backoff-delay", "250",
+                "camel.routecontroller.backoff-delay", "1000",
                 "camel.routecontroller.backoff-max-attempts", "10",
                 "camel.routecontroller.unhealthy-on-restarting", "true",
                 "camel.routecontroller.unhealthy-on-exhausted", "false");

